docs/modules/objectives/scoring.mdx

```diff
@@ -20,6 +20,14 @@ It can also be used to create score boxes that give a player or team a certain a
   | `<box>` | A single score box. | [Score Box](#score-boxes) |
 </div>
 
+##### Score Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
+</div>
```

Comment:
```suggestion
| Attribute | Description | Value |
|---|---|---|
| `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
```